ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/services/DefaultKsqlClient.java

```diff
@@ -225,4 +233,11 @@ private static Map<String, String> toClientProps(final Map<String, Object> confi
     }
     return clientProps;
   }
+
+  private long getQueryTimeout(final Map<String, ?> configOverrides) {
+    if (configOverrides.containsKey(KsqlConfig.KSQL_QUERY_PULL_FORWARDING_TIMEOUT_MS_CONFIG)) {
+      return (Long) configOverrides.get(KsqlConfig.KSQL_QUERY_PULL_FORWARDING_TIMEOUT_MS_CONFIG);
+    }
+    return ksqlConfig.getLong(KsqlConfig.KSQL_QUERY_PULL_FORWARDING_TIMEOUT_MS_CONFIG);
```

Comment:
shouldnt this just be returning `KsqlConfig.KSQL_QUERY_PULL_FORWARDING_TIMEOUT_MS_DEFAULT` if no override is found

Reply:
If you do that, you won't be able to set this value in the config file.  If you do `ksqlConfig.getLong(KsqlConfig.KSQL_QUERY_PULL_FORWARDING_TIMEOUT_MS_CONFIG);`, it will attempt to find it in the config file, and if not, default to `KSQL_QUERY_PULL_FORWARDING_TIMEOUT_MS_DEFAULT`.

Reply:
ah okay i see, makes sense!